exporter/debugexporter/internal/normal/profiles.go

```diff
@@ -35,10 +36,15 @@ func (normalProfilesMarshaler) MarshalProfiles(pd pprofile.Profiles) ([]byte, er
 				buffer.WriteString(" samples=")
 				buffer.WriteString(strconv.Itoa(profile.Sample().Len()))
 
-				if profile.Attributes().Len() > 0 {
-					profileAttributes := writeAttributes(profile.Attributes())
+				if profile.AttributeIndices().Len() > 0 {
+					attrs := []string{}
+					for _, i := range profile.AttributeIndices().AsRaw() {
+						a := profile.AttributeTable().At(int(i))
+						attrs = append(attrs, fmt.Sprintf("%s=%s", a.Key(), a.Value().AsString()))
```

Comment:
Adding the space in front of the format string (` %s=%s`) avoids having a space+linefeed at the end.

Reply:
I'd rather not refactor this, to keep what exactly was changed readable.